chain/blockValidator.go

```diff
@@ -123,8 +123,10 @@ func TransactionCheck(ctx context.Context, block *block.Block) error {
 			return fmt.Errorf("transaction sanity check failed: %v", err)
 		}
 		if err := bvs.VerifyTransactionWithBlock(txn, block.Header); err != nil {
+			bvs.Reset()
```

Comment:
Do we need this Reset at all? bvs is newly created and is never used

Reply:
mainly for consistency sake, doesn't hurt anyway 😂